### ISSUE TITLE
loss = 0,train acc = 0

### ISSUE THREAD

**Issue by duynguyen5896** [astorfi/3D-convolutional-speaker-recognition#25]
Hi astorfi, 
I'm trying to use your code to train a model with 31 labels, 60 samples for each label. However, when i use train_softmax.py, last minibatches return loss = 0 while train acc = 0. Do you have any idea to fix it?

![image](https://user-images.githubusercontent.com/34588978/41216974-a3752116-6d80-11e8-8266-de2eef7fb431.png)

Thank you.

**Comment by astorfi** [comment]
@duynguyen5896 train_softmax.py is just for training all the layers as you mentioned. It is created for generating the background model. It is hard to say why it's not converging because I do not have access to your data or the details.

**Comment by duynguyen5896** [comment]
@astorfi, for the first problem, i realized that my label was higher than 31 and after relabeled from 0 to 30, it works. Thank for your answer, it helps me a lot.

**Comment by astorfi** [comment]
@duynguyen5896 Good to hear that.

**Comment by duynguyen5896** [comment]
@astorfi , can you tell me how many samples per person you used to train and validate your background model?

**Comment by astorfi** [comment]
Please refer to [this issue](https://github.com/astorfi/3D-convolutional-speaker-recognition/issues/26) for futher details.

**Comment by duynguyen5896** [comment]
Thank for your help again @astorfi !!!

**Comment by astorfi** [comment]
@duynguyen5896 My pleasure
